Update-to-Milestone-2-MyPortfolio-Project

fernando-silvabr66 opened the pull request:
Update after revision of layout based on Figma web page, as part of the Day 4 - Week 1 - Activity at Microverse.

fernando-silvabr66 committed:
Add linters configuration to the project

fernando-silvabr66 committed:
Add .gitignore file and update README.md

fernando-silvabr66 committed:
Add initial index.html and style.css files to project

fernando-silvabr66 committed:
Add image files to project

fernando-silvabr66 committed:
Update index.html ans style.css files for Skills Section

fernando-silvabr66 committed:
Add linters cfg files to project

fernando-silvabr66 committed:
Update .gitignore file

fernando-silvabr66 committed:
New round of layout adjustments to index.html and Style.css files

fernando-silvabr66 committed:
Update on style.css file after linters run

Hamzaoutdoors commented on README.md:
```diff
@@ -1,2 +1,57 @@
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- The `MIT` link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

fernando-silvabr66 replied:
Thanks for the heads up.

Hamzaoutdoors commented on index.html:
```diff
@@ -0,0 +1,245 @@
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```
- [x] Please kindly fix this image `src`, as you can notice in the screenshot below the image is broken, the `src` is src="./images/background.jpg" with one point to track the path for your image

![Capture](https://user-images.githubusercontent.com/80895497/148665633-79dad873-f1e6-4225-941c-e2138a6b9ff2.PNG)

Hamzaoutdoors commented on index.html:
```diff
@@ -0,0 +1,245 @@
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```
The same bug requested in the comment below, Please, fix the image problem by changing the `src` to `src="./images/background.jpg"`

fernando-silvabr66 replied:
Fixed also... thanks.

Hamzaoutdoors reviewed (changes requested):
Hey @fernando-silvabr66, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fernando-silvabr66 committed:
Add MIT.md and updated app_screenshot.png files + code-review corrections

Add MIT.md and app_screenshot.png files.
Applied corrections of the 1st Code Review.

Cathella reviewed (changes requested):
## 🟨 Status: Changes Required 🟥 

### Hi @fernando-silvabr66 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As pointed out by the previous reviewer, the project images do not show up in the live demo, please refer to the previous reviewers comments, fix the problem and let your portfolio shine 🌞 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fernando-silvabr66 commented:
Hi, I have changed the src configuration for the images  (it's appearing in
my Live server).
Do you have any additional suggestions (please)?

Regards.

Em sáb., 8 de jan. de 2022 23:09, Nakitto Catherine <
***@***.***> escreveu:

> ***@***.**** requested changes on this pull request.
> 🟨 Status: Changes Required 🟥 Hi @fernando-silvabr66
> <https://github.com/fernando-silvabr66> 👋🏻,
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
> Required Changes ♻️
>
> *Check the comments under the review.*
>
>    - As pointed out by the previous reviewer, the project images do not
>    show up in the live demo, please refer to the previous reviewers comments,
>    fix the problem and let your portfolio shine 🌞
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fernando-silvabr66/Milestone-2-MyPortfolio/pull/1#pullrequestreview-847208374>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASS42Y6CFQ74Z6BLV56IPV3UVDU6JANCNFSM5LRHUZ4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> .com>
>

fernando-silvabr66 commented:
Sorry fot the mistyping... word corrector...

Em sáb., 8 de jan. de 2022 23:15, Fernando Silva <
***@***.***> escreveu:

> Hi, i have chances the src configuração for the images  (it's appearing in
> my Live server).
> Do you have any adicional suggestion(please)?
>
> Regards.
>
> Em sáb., 8 de jan. de 2022 23:09, Nakitto Catherine <
> ***@***.***> escreveu:
>
>> ***@***.**** requested changes on this pull request.
>> 🟨 Status: Changes Required 🟥 Hi @fernando-silvabr66
>> <https://github.com/fernando-silvabr66> 👋🏻,
>>
>> Great work on making the changes requested by a previous reviewer 👏
>> You've done well implementing some of the requested changes, but there
>> are still some which aren't addressed yet.
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>>
>>    - As pointed out by the previous reviewer, the project images do not
>>    show up in the live demo, please refer to the previous reviewers comments,
>>    fix the problem and let your portfolio shine 🌞
>>
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
>> PR. However, I strongly recommend you to take them into account as they can
>> make your code better. Some of them were simply missed by the previous
>> reviewer and addressing them will really improve your application.*
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/fernando-silvabr66/Milestone-2-MyPortfolio/pull/1#pullrequestreview-847208374>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASS42Y6CFQ74Z6BLV56IPV3UVDU6JANCNFSM5LRHUZ4Q>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you were mentioned.Message ID:
>> <fernando-silvabr66/Milestone-2-MyPortfolio/pull/1/review/847208374@
>> github.com>
>>
>

fernando-silvabr66 committed:
Applied more changes to src for images corrections.

fernando-silvabr66 commented:
Due to time restrictions, I used an alternative solution in order to have it submitted in time... thanks.

Cathella reviewed (approved):
## 🟩 Status: Project Approved 🎉 

### Hi @fernando-silvabr66 👋🏻, 

Good job making all the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes ♻️ 

- [ ] [OPTIONAL] Please try to center the rest of the images too so that you have a consistent design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fernando-silvabr66 commented:
Finalized... cheers!
